edb/common/debug.py

```diff
@@ -141,6 +142,26 @@ def print(*args):
     builtins.print(*args)
 
 
+def enable_typechecks():
+    if 'edb.common.checked' in sys.modules:
```

Comment:
I'd just import these modules as the overhead is low. Otherwise seems very fragile -- what if something changes in how we load the system and enable_typechecks() is called before these imports.

Reply:
The modules themselves enable or disable type checking on import based on what `debug.flags.typecheck` is saying. Because of that, they import `edb.common.debug`.

Therefore, we cannot indiscriminately import here if the module wasn't imported before. Otherwise during import of `edb.common.debug` we will end up importing `edb.common.checked` which in turn will end up importing `edb.common.debug`. An import cycle.

But to have `--devmode` enable runtime typechecks by default, we *do* need to update the status of runtime typechecking using `edb.common.debug.init_debug_flags()`. This is because by the time `edb.tools.edb.edbcommands()` is called, `edb.common.checked` and `edb.common.ast.base` were already imported and type checks were disabled.

Reply:
Got it.